### DISCUSSION TITLE
Souci reconnaissante scolarité CANVA

### DISCUSSION THREAD

**Discussion by JspStu** [community/community#51856] [GitHub Education] [answered]
I have a problem for the CANVA app which is free for students. I sent a photo of my school card and my school certificate, but my diploma is 100% digital, so remotely via video. I live in the south of France, and my school is in Paris. So Github does not accept my request for CANVA

**Comment by samus-aran** [comment, marked as answer]
Hi @JspStu  👋 ! 

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers).
